app/views/register-with-a-gp/donor-organs.html

```diff
@@ -0,0 +1,92 @@
+{% extends 'templates/nhs_transaction_layout.html' %}
+{% import 'includes/form_macros.html' as form_macros %}
+
+{% block afterHeader %}
+  {% include "includes/service-headers/register-with-a-gp.html" %}
+{% endblock %}
+
+{% block content %}
+
+<main id="content" role="main">
+
+  <h1 class="heading-xlarge">
+    Organ donation
+  </h1>
+
+  <form action="" method="get" class="form text">
+
+    <fieldset class="form-group">
+      <legend class="form-label-bold">Are you willing to donate your organs for transplantation after your death?</legend>
+      <input type="radio" id="organs-yes" name="organs">
+      <label class="form-control-radio js-toggle" for="organs-yes" data-show="#organ-donor">
+        Yes
+      </label>
+      <input type="radio" id="organs-no" name="organs">
+      <label class="form-control-radio js-toggle" for="organs-no" data-hide="#organ-donor">
+        No
+      </label>
+    </fieldset>
+
+    <div class="js-hidden panel-indent" id="organ-donor">
+      <fieldset>
+        <legend>I will donate</legend>
+
+        <div class="form-group">
+          <input type="checkbox" id="corneas">
+          <label class="form-control-checkbox" for="corneas" tabindex="0" role="checkbox">
+            Corneas
+          </label>
+        </div>
+
+        <div class="form-group">
+          <input type="checkbox" id="heart">
+          <label class="form-control-checkbox" for="heart">
+            Heart
+          </label>
+        </div>
+
+        <div class="form-group">
+          <input type="checkbox" id="kidneys">
+          <label class="form-control-checkbox" for="kidneys">
+            Kidneys
+          </label>
+        </div>
+
+        <div class="form-group">
+          <input type="checkbox" id="liver">
+          <label class="form-control-checkbox" for="liver">
+            Liver
+          </label>
+        </div>
+
+        <div class="form-group">
+          <input type="checkbox" id="lungs">
+          <label class="form-control-checkbox" for="lungs">
+            Lungs
+          </label>
+        </div>
+
+        <div class="form-group">
+          <input type="checkbox" id="pancreas">
+          <label class="form-control-checkbox" for="pancreas">
+            Pancreas
+          </label>
+        </div>
+
+        <div class="form-group">
+          <input type="checkbox" id="any-part-of-my-body">
```

Comment:
We should review the overall content with Dr Marcus. But yes, we'll preselect this, good point.